src/scripts/ordo_report_mapping_annotations/report_mapping_annotations.py

```diff
@@ -0,0 +1,144 @@
+"""Report ORDO mapping strings
```

Comment:
I would prefer to review this file in person

Reply:
Alright, sounds good. I'll make a note to review these in our next meeting.